library/conn_bridge.c

```diff
@@ -412,3 +416,71 @@ void on_input(uv_stream_t *s, ssize_t len, const uv_buf_t *b) {
         }
     }
 }
+
+int conn_bridge_info(ziti_connection conn, char *buf, size_t buflen) {
+    if (conn == NULL || !conn->bridged) {
+        return ZITI_INVALID_STATE;
```

Comment:
does this deserve a log message?

Reply:
no, it is an internal information function -- this return code just means the connection is not bridged